### ISSUE TITLE
per-user licensing

### ISSUE THREAD

**Issue by lightbulblighter** [lightbulblighter/Rboxlo#68]
builds upon laciwids in some way

**Comment by lightbulblighter** [comment]
OK, SO as of the commit that's been lurking in my hard-drive that I won't push just yet(!):
- LACIWID implementation complete. Glorified OTPs that live for 20 minutes before being pruned. Generated and returned upon License validation, and is not generated FROM a License. Again, returned from the server, not generated from the client. As it is a simple implementation, I doubt that it has anything that will change unless I restructure the License part entirely.
- License implementation complete. Licenses are basically keys that last forever that bar off a given ClientApplication. This implementation as it stands is insecure. Perhaps something with hardware information can help. Nonetheless, this implementation is subject to change.

Per-user licensing, as this issue so vaguely states has been implemented sort-of. I just made applications not use a global license. Why was I doing that, anyway?

We may be able to learn from the OTP implementation on how to properly implement this. I can see a number of gaping security holes in this. As Licenses will be inevitably used to protect private ClientApplications they should be as secure as possible.
